### ISSUE TITLE
Unhandled JS exception in main.js:1068 in IE11

### ISSUE THREAD

**Issue by michaelklishin** [rabbitmq/rabbitmq-management#98]
JS code around that line:

``` javascript
                 if (matches != null && matches.length > 1) {
                     for (item of contexts) {
                         if (matches[1].indexOf(item) == 0) {update_pages(item, 1)};
                     }
                 }
```

I'm not a JS exprt but the `for(i of items)` construct does look non-standard to me.

**Comment by michaelklishin** [comment]
Reported by @Michael-Ekkert.

Does the same happen in Chrome on Windows?

`for … of` does seem to be [fairly recent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of). Perhaps `Array.prototype.forEach` should be used instead.

**Comment by michaelklishin** [comment]
Mozilla's docs don't even mention EI for `for … of`. Looks like Edge is the first browser to support it. So a workaround is to use Chrome or Edge.

**Comment by Michael-Ekkert** [comment]
It works in chrome, just not IE11 which is company standard.

Thanks

Sent from my iPhone

On Dec 29, 2015, at 6:25 PM, "Michael Klishin" <notifications@github.com<mailto:notifications@github.com>> wrote:


Mozilla's docs don't even mention EI for for … of. Looks like Edge is the first browser to support it. So a workaround is to use Chrome or Edge.

**Comment by michaelklishin** [comment]
Sure, we didn't intend to break IE compatibility.

**Comment by michaelklishin** [comment]
Fixed in #99, will be in `3.6.1`.
